providers/explainshell/explainshell.go

```diff
@@ -0,0 +1,30 @@
+package explainshell
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/zquestz/s/providers"
+)
+
+// strings.Replace will apply for all matches when n < 0
+const REPLACE_ALL int = -1
```

Comment:
Please use camelcase for constants and place them in a const block. Also only export them if they are required by other packages. Should look like:

```
const (
  replaceAll int = 1
  providerName = "explainshell"
  providerURL = "https://explainshell.com
)
```